src/main/java/step2/domain/LottoMarket.java

```diff
@@ -0,0 +1,61 @@
+package step2.domain;
+
+import java.util.HashMap;
+import java.util.Map;
+
+public class LottoMarket {
+
+    private final int PRICE = 1000;
+    private LottoMachine machine;
```

Comment:
```suggestion
    private static final int PRICE = 1000;
    
    private LottoMachine machine;
```

상수에 static을 붙여주는 이유에 대해 고민해보아요
또한 클래스 변수와 인스턴스 변수 사이에 공백을 추가한다면 가독성이 상승합니다!